core/io.rbs

```diff
@@ -2278,7 +2278,7 @@ class IO < Object
   # potential security vulnerabilities if called with untrusted input; see
   # [Command Injection](rdoc-ref:command_injection.rdoc).
   #
-  def self.binread: (String name, ?Integer? length, ?Integer offset) -> String
+  def self.binread: (_ToPath | string name, ?Integer? length, ?Integer offset) -> String
```

Comment:
We can use  builtin type alias `path` instead of `_ToPath | string`.
https://github.com/ruby/rbs/blob/b51d7ded7d901051cbb6f86fa78974ddd15c00d5/core/builtin.rbs#L255

Reply:
Oh, nice. Thanks.